modules/gltf/src/lib/types/gltf-types.ts

```diff
@@ -54,6 +54,14 @@ export type GLTF_KHR_draco_mesh_compression = {
   extras?: any;
 };
 
+export type GLTFHeader = {
```

Comment:
Nit: Naming - This is not part of glTF spec, it is part of the 3DTILES spec. Isn't this in fact `BatchTable` above?

Reply:
Moved to 3dTiles type file.